### PR TITLE
Slight UI fixes.

### PR DESCRIPTION
**Pull request by davinchia**
## What
- Fix a broken link and some bad text. (More details in line)
- Remove out of date documentation. (Up to date documentation is [here](https://github.com/airbytehq/airbyte/blob/master/docs/faq/technical-support.md))

## Pre-merge Checklist
- [ ] *Run integration tests*

### CODE REVIEW

File: airbyte-webapp/src/locales/en.json [hunk 1]
```diff
@@ -112,7 +112,7 @@
   "sources.schemaSelectAll": "select all",
   "sources.schemaUnselectAll": "unselect all",
   "sources.settings": "Settings",
-  "sources.syncHistory": "Sync history",
+  "sources.syncHistory": "Sync History",
```
Comment (davinchia) [inline airbyte-webapp/src/locales/en.json:115]: the lower-case h was really bothering me:

Screen Shot 2021-03-11 at 7 39 36 PM

**Comment by davinchia** [comment]
@Jamakase would appreciate if if you check to make sure I made the changes in the right spot!

File: docs/technical-support.md [hunk 2]
```diff
@@ -1,26 +0,0 @@
----
```
Comment (sherifnada) [inline docs/technical-support.md:1]: was this a dupe?

**Review by sherifnada — APPROVED** [review]

File: docs/technical-support.md (hunk 2 above)
Comment (davinchia) [reply docs/technical-support.md:1]: poked around git a bit and best guess is `technical-support` was originally here and later moved under the `faq` section. the doc was left hanging when that happened.

**Review by cgardens — APPROVED** [review]

**Pull request merged**